integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeBlockchainApi.kt

```diff
@@ -75,8 +76,9 @@ open class CrowdNodeBlockchainApi @Inject constructor(
         val acceptTx = paymentService.sendCoins(crowdNodeAddress, requestValue, selector)
         log.info("acceptTx id: ${acceptTx.txId}")
         val errorResponse = CrowdNodeErrorResponse(params, requestValue)
-        val tx = walletDataProvider.observeTransactions(
+        val tx = walletData.observeTransactions(
             CrowdNodeWelcomeToApiResponse(params),
+            PossibleWelcomeResponse(walletData.transactionBag, accountAddress),
```

Comment:
It's easier to add these as separate filters.
If/when we implement the better solution, `possible` filters will go away from here, but probably will have to stay in other places since some users might have valid accounts that are only detected with these filters.